### PR TITLE
Release notes update for CSM Replication

### PR DESCRIPTION
**Pull request by mjsdell**
# Description
Updating the release notes with new known issues reported for CSM Replication

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/517 |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by AronAtDell — APPROVED** [review]

File: content/docs/replication/release/_index.md [hunk 1]
```diff
@@ -23,4 +23,8 @@ Description: >
 
 ### Known Issues
 
-There are no known issues in this release.
+| Github ID | Description  |   
+|-----------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [514](https://github.com/dell/csm/issues/514) | **PowerScale:** When creating a replicated PV in PowerScale, the replicated PV's AzServiceIP property has the target PowerScale endpoint instead of the one defined in the target Storage class.                            |   
+| [515](https://github.com/dell/csm/issues/515) | **PowerScale:** If you failover with an application still running and having a mounted the volume, on the target site we cannot mount the PVC due to : "mount.nfs: Stale file handle".                                      |
```
Comment (rsedlock1958) [inline content/docs/replication/release/_index.md:29]: Line 29: Rewrite: If you failover with an application still running and the volume mounted on the target site, then we cannot mount the PVC due to : "mount.nfs: Stale file handle".

File: content/v1/replication/release/_index.md [hunk 2]
```diff
@@ -23,4 +23,8 @@ Description: >
 
 ### Known Issues
 
-There are no known issues in this release.
+| Github ID | Description  |   
+|-----------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [514](https://github.com/dell/csm/issues/514) | **PowerScale:** When creating a replicated PV in PowerScale, the replicated PV's AzServiceIP property has the target PowerScale endpoint instead of the one defined in the target Storage class.                            |   
+| [515](https://github.com/dell/csm/issues/515) | **PowerScale:** If you failover with an application still running and having a mounted the volume, on the target site we cannot mount the PVC due to : "mount.nfs: Stale file handle".                                      |
```
Comment (rsedlock1958) [inline content/v1/replication/release/_index.md:29]: Line 29: Rewrite: "If you failover with an application still running and the volume mounted on the target site, then we cannot mount the PVC due to : "mount.nfs: Stale file handle".

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. There are a couple of rewrites required.

File: content/docs/replication/release/_index.md (hunk 1 above)
Comment (mjsdell) [reply content/docs/replication/release/_index.md]: fixed

File: content/v1/replication/release/_index.md (hunk 2 above)
Comment (mjsdell) [reply content/v1/replication/release/_index.md]: fixed

**Pull request merged**